### PR TITLE
clementine: improve examples, more info link, and add example

### PR DESCRIPTION
**Pull request by vitorhcl**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):** 1.4 rc2

### CODE REVIEW

File: pages/common/clementine.md [hunk 1]
```diff
@@ -19,18 +19,18 @@
...
+- Skip to the next or previous track:
 
-`clementine --next`
+`clementine --{{next|previous}}`
 
-- Skip to the previous track:
+- Create a new playlist with one or more music files or URLs:
 
-`clementine --previous`
+`clementine --create {{url_or_path/to/music1.ext url_or_path/to/music2.ext ...}}`
```
[the remark this answers is not in the document]
Comment (vitorhcl) [reply pages/common/clementine.md:28]: I think the pipe syntax is better for placeholders that accept a single argument (more clear and shorter) and the underscore syntax is better for list placeholders (uses the number a single time, not repeatinf it unnecessarily).

I don't see it as a inconsistency, but like "using the right tool for the job"

File: pages/common/clementine.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
-> More information: <https://www.clementine-player.org>.
+> More information: <https://github.com/clementine-player/Clementine/wiki>.
 
-- Open Clementine:
+- Start the GUI or bring it to front:
 
 `clementine`
 
-- Start playing a music file:
+- Start playing a music:
```
[the remark this answers is not in the document]
Comment (vitorhcl) [reply pages/common/clementine.md:10]: I think we can leave it because of the file

[the remark this answers is not in the document]
Comment (vitorhcl) [reply pages/common/clementine.md:10]: Maybe "an audio"?

**Review by blueskyson — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**